### DISCUSSION TITLE
Welcome to quicktype Discussions!

### DISCUSSION THREAD

**Discussion by dvdsgl** [quicktype/quicktype#1607] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joelguittet** [comment]
Hello @dvdsgl @schani and everybody

This project is really cool. But I wonder about one thing: is it still maintained ? What about the PR which are submitted ?

The number of issues opened with no answer (I have asked myself one point because I'm working on cJSON generator) is very important and plenty of them remains with no answer. Some are duplicated. Some are just written without details and should probably be closed.

Regards,
Joel

**Reply by schani** [reply]
Hi Joel,

Unfortunately we don't have the time to maintain quicktype anymore.  I try to go through the PRs every now and then and help a bit, but it's clearly not enough.

Mark

**Reply by joelguittet** [reply]
Hello Mark,

Thanks for the quick response to this. Really annoying to see it has maybe so much success that it is an overhead for you.
Do you know similar projects to convert JSON schema to source code in various languages ?
I appreciate the job done here but if contributions can't be integrated I'm not sure if I will finish it.

Joel

**Reply by schani** [reply]
I don't know of any projects of similar quality.

What's the feature you're working on?

**Reply by joelguittet** [reply]
Me too. My single reference is https://json-schema.org/implementations.html and quicktype looks the better solution today :-)

Actually generating C code to manipulate JSON objects. I'm used to work with cJSON. I have started creating a generator but I'm faced to an issue for arrays (https://github.com/quicktype/quicktype/issues/1984)

**Reply by bzuidgeest** [reply]
> Hi Joel,
> 
> Unfortunately we don't have the time to maintain quicktype anymore. I try to go through the PRs every now and then and help a bit, but it's clearly not enough.
> 
> Mark

Well that explains my PR just sitting there :( . Any chance of a return to the project in the future? At least among free tools, I found nothing better.

**Reply by fwal** [reply]
> Hi Joel,
> 
> Unfortunately we don't have the time to maintain quicktype anymore. I try to go through the PRs every now and then and help a bit, but it's clearly not enough.
> 
> Mark

Sad to hear that as it's a really useful and cool project, you've done an amazing job 👏🏻
Have you considered adding maintainers from the community? It would be a shame to let the project wither away as, at least by judging on the Dart-related issues, there's a lot of passion for the project out there 😅

**Reply by dvdsgl** [reply]
We've recently put the project back on its feet by fixing CI.
